src/main/java/org/usfirst/frc4904/robot/commands/vision/VisionAlignToWall.java

```diff
@@ -0,0 +1,32 @@
+package org.usfirst.frc4904.robot.commands.vision;
+
+import java.util.List;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.commands.chassis.SimpleSplines;
+
+import edu.wpi.first.wpilibj.geometry.Pose2d;
+import edu.wpi.first.wpilibj.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.trajectory.Trajectory;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+/**
+ * Given distance, beta, theta from target, drive right to the target
+ */
+public class VisionAlignToWall extends SequentialCommandGroup {
+    protected static final double DEFAULT_ANGLE = 0.0;
+    protected static final double DEFAULT_DISTANCE = 0.0;
+
+    public VisionAlignToWall() {
+        double beta = RobotMap.NetworkTables.Vision.beta.getDouble(DEFAULT_ANGLE);
+        double deltaAngle = beta - 90;
```

Comment:
All angles are in radians.

Reply:
Also, if the robot is directly facing the target, `beta` should be 0. Wouldn't that mean that `deltaAngle` should also be 0?